amf-client/shared/src/test/resources/org/raml/parser/fragments/library/output.txt

```diff
@@ -7,6 +7,7 @@ types:
     additionalProperties: true
```

Comment:
why isn't this a yaml/raml file?

Reply:
To be honest, I don't remember. About 3 years ago, when we integrated the old raml parser we've chosen that kind of file. All the suit works that way, maybe we can talk about the suit, but is not in the scope of this change.